### PR TITLE
Fix services and international link for header and alignment for search

### PR DESCRIPTION
**Pull request by webbyfox**
This changeset adds some style fix for mobile and desktop device along with some link fixes

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/2326
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.
- [x] Includes screenshot(s) - ideally before and after, but at least after
- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

image

image

### CODE REVIEW

File: core/templates/components/header_footer/search.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div id="great-header-search-wrapper" class="great-header-search-wrapper">
-  <div class="great-header-search-container">
+<div id="great-header-search-wrapper" class="great-header-search-wrapper ">
```
Comment (stevejalim) [inline core/templates/components/header_footer/search.html:1]: Is that added extra space needed here?

File: core/templates/components/header_footer/domestic_header_nav.html [hunk 2]
```diff
@@ -38,7 +38,7 @@
       </li>
 
       <li class="menu-item" data-js-menu-item-container>
-        <a href="{{ header_footer_urls.services }}" class="link-heading" id="header-services-{{ device_type }}" {% if js_tag %}data-js-menu-index=6{% endif %}>Services</a>
+        <a href="/services/" class="link-heading" id="header-services-{{ device_type }}" {% if js_tag %}data-js-menu-index=6{% endif %}>Services</a>
```
Comment (stevejalim) [inline core/templates/components/header_footer/domestic_header_nav.html:41]: hard-coded /services/ is OK but we could also use {% slugurl 'services' %} - am easy either way

**Review by stevejalim — APPROVED** [review]
LGTM with a couple of small comments

File: core/templates/components/header_footer/search.html (hunk 1 above)
Comment (webbyfox) [reply core/templates/components/header_footer/search.html]: yeah - good catch - add/removing class has left that space

File: core/templates/components/header_footer/domestic_header_nav.html (hunk 2 above)
Comment (webbyfox) [reply core/templates/components/header_footer/domestic_header_nav.html]: ok - added slugurl

**Pull request merged**